webapp/store/country/hooks/useCountries.js

```diff
@@ -0,0 +1,27 @@
+import { Area } from '@common/country'
+import { useI18n } from '@webapp/components/hooks'
+import { useDispatch, useSelector } from 'react-redux'
+import * as CountryActions from '@webapp/store/country/actions'
+import * as CountryState from '@webapp/app/country/countryState'
+import { useEffect } from 'react'
+
+export const sortCountries = (countries, i18n) => {
+  const _compareListName = Area.getCompareListName(i18n)
+  const _compareCountries = (country1, country2) => _compareListName(country1.countryIso, country2.countryIso)
+
+  return Object.fromEntries(
+    Object.entries(countries).map(([role, roleCountries]) => [role, [...roleCountries].sort(_compareCountries)])
+  )
+}
+
+export const useCountries = () => {
```

Comment:
dont we have this in App Store ?

Reply:
This is different, see issue comment

Reply:
why don't you iterate over the countries we have in the store and get the role from those?